Kudu.Core.Test/Jobs/ScheduleFacts.cs

```diff
@@ -60,8 +68,15 @@ public void GetNextInterval_NextAfterDST_ReturnsExpectedValue()
 
             TimeSpan interval = schedule.GetNextInterval(now, ignoreMissed: true);
 
+            var expectedHours = new TimeSpan( 168 , 0 , 0 );
```

Comment:
nit: expectedInterval

Reply:
Also, you can remove the comment above "Note: this test uses Local time..." now that you've fixed this :) Unfortunately it looks like I was aware of this issue when I checked the code in originally, but I was focused on our Redmond TimeZone and not yours :)